st2common/st2common/hooks.py

```diff
@@ -13,16 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import datetime
+import urlparse
```

Comment:
On a related note - we should also start running tests with Python 3 (if possible, dunno about dependencies).